pkg/sdkserver/datasets.go

```diff
@@ -12,25 +12,21 @@ import (
 
 type datasetRequest struct {
 	Input           string `json:"input"`
-	Workspace       string `json:"workspace"`
```

Comment:
We don't need to rely on this anymore, since the workspace provider takes care of everything.